src/ciphers/sm4.c

```diff
@@ -67,7 +67,7 @@ static const sm4_u8_t sbox_table[16][16] = {
  * S-box
  * defined in section 2.6 S-box
  */
-LTC_INLINE static sm4_u8_t sbox(sm4_u8_t a)
+LTC_INLINE static sm4_u8_t s_sbox(sm4_u8_t a)
```

Comment:
I'd suggest to simply namespace all the functions, but leave their original name intact.

```suggestion
LTC_INLINE static sm4_u8_t s_sm4_sbox(sm4_u8_t a)
```

Reply:
done